src/reducer.js

```diff
@@ -1,18 +1,22 @@
-import { INITIAL_STATE, guess, setActivePerson, clearGuess, hint, clearHint } from './core';
+import { INITIAL_STATE, guess, setActivePerson, clearGuess, hint, clearHint, startLoading, endLoading } from './core';
 
 export default function reducer(state = INITIAL_STATE, action) {
   switch (action.type) {
-  case 'SET_ACTIVE_PERSON':
-    return setActivePerson(state, action.person);
-  case 'GUESS':
-    return guess(state, action.guess);
-  case 'CLEARGUESS':
-    return clearGuess(state);
-  case 'HINT':
-    return hint(state);
-  case 'CLEARHINT':
-    return clearHint(state);
-  default:
-    return state;
+    case 'SET_ACTIVE_PERSON':
+      return setActivePerson(state, action.person);
+    case 'GUESS':
+      return guess(state, action.guess);
+    case 'CLEARGUESS':
+      return clearGuess(state);
+    case 'HINT':
+      return hint(state);
+    case 'CLEARHINT':
+      return clearHint(state);
+    case 'START_LOADING':
+      return startLoading(state);
+    case 'END_LOADING':
+      return endLoading(state);
+    default:
+      return state;
```

Comment:
There's a lot of refactoring going on, which somewhat obscures the relevant changes. In general, I suggest doing one refactor or change per commit, so maybe string quote changes (`"` -> `'`) in one commit, whitespace changes in another, and then the loading workflow in a third.